eng/MSBuild/ProjectStaging.props

```diff
@@ -0,0 +1,26 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- Amend the description based on stage -->
+    <Description Condition="'$(Stage)' == 'dev'">Experimental package. $(Description)</Description>
+    <Description Condition="'$(Stage)' == 'obsolete'">Obsolete Package. $(Description)</Description>
+
+    <!--
+        When DotNetFinalVersionKind is set to 'release', this branch will produce stable outputs for 'Shipping' packages
+
+        In 8.0, we shipped Microsoft.AspNetCore.Testing package as stable even when it was in the dev stage, so we
+        keep it as stable for compatiblity.
+      -->
+    <_IsStable Condition="('$(Stage)' != 'dev' and '$(Stage)' != 'preview') Or '$(MSBuildProjectName)' == 'Microsoft.AspNetCore.Testing'">true</_IsStable>
+    <DotNetFinalVersionKind Condition=" '$(DotNetFinalVersionKind)' == '' And '$(_IsStable)' == 'true' ">release</DotNetFinalVersionKind>
+
+    <!-- Preview packages: do not use stable branding and do not warn about lack of [Experimental] -->
+    <NoWarn Condition="'$(Stage)' == 'dev' or '$(Stage)' == 'preview'">$(NoWarn);LA0003</NoWarn>
+    <!--
+        Makes it such that the package version won't be stabilized even when the rest of the repo is going stable.
+        https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Versioning.md#package-version
+      -->
+    <SuppressFinalPackageVersion />
+    <SuppressFinalPackageVersion Condition=" '$(configureDotNetFinalVersionKind)' == 'true' And '$(_IsStable)' != 'true' ">true</SuppressFinalPackageVersion>
```

Comment:
Is `configureDotNetFinalVersionKind` right? Or did you mean `DotNetFinalVersionKind`?

Reply:
Ok I do see you did mean the right property based on the pipeline changes, but can you help me understand what is this for? I'm not sure I'm following as we tipically wouldn't want to set this globally.

Reply:
Boo, I copied the wrong property name. I'll fix it, thank you.

Reply:
This property is only set whenever we're producing a signed build, which excludes the main and pull-requests. We don't want to stabilise for any other branch, right?